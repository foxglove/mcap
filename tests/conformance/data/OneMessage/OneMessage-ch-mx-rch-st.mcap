version https://git-lfs.github.com/spec/v1
oid sha256:deae57fbc576edf021df07f03f28e949c524ee157ef70cca9711ecdd79b6ae48
size 382
