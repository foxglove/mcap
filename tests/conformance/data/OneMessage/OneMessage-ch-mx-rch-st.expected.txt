Header library="" profile=""
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema=b schema_encoding=c schema_name=Example topic=example
Message channel_id=1 log_time=2 message_data=<010203> publish_time=1 sequence=10
MessageIndex channel_id=1 records={2=66}
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema=b schema_encoding=c schema_name=Example topic=example
Statistics attachment_count=0 channel_count=1 channel_message_counts={1=1} chunk_count=1 message_count=1
ChunkIndex chunk_length=139 chunk_start_offset=25 compressed_size=100 compression="" end_time=2 message_index_length=31 message_index_offsets={1=166} start_time=2 uncompressed_size=100
Footer summary_offset_start=0 summary_start=210
